homeassistant/components/heos/media_player.py

```diff
@@ -387,6 +387,15 @@ async def async_play_media(
             await self._player.play_preset_station(index)
             return
 
+        if media_type == "queue":
+            # media_id must be an int
+            try:
+                queue_id = int(media_id)
+            except ValueError:
+                raise ValueError(f"Invalid queue id '{media_id}'") from None
```

Comment:
I am wondering if it is better to raise `HomeAssistantError` with a translatable error

Reply:
Yeah, the decorator `catch_action_error` does that translation. In the test below, you'll see it's raising `HomeAssistantError`